docs/src/main/java/org/geotools/po/bindings/USAddressBinding.java

```diff
@@ -24,49 +36,58 @@
  *      &lt;/xsd:sequence&gt;
  *      &lt;xsd:attribute fixed="US" name="country" type="xsd:NMTOKEN"/&gt;
  *  &lt;/xsd:complexType&gt; 
- *		
- *	  </code>
- *	 </pre>
+ *        
+ *      </code>
+ *     </pre>
  * </p>
  *
  * @generated
  */
 public class USAddressBinding extends AbstractComplexBinding {
 
-	ObjectFactory factory;		
-	public USAddressBinding( ObjectFactory factory ) {
-		super();
-		this.factory = factory;
-	}
+    ObjectFactory factory;        
+    public USAddressBinding( ObjectFactory factory ) {
+        super();
+        this.factory = factory;
+    }
+
+    /**
+     * @generated
+     */
+    public QName getTarget() {
```

Comment:
We are getting a lot of conflicts here, but the code looks the same.  Do we have whitespace conflict I am not seeing? The answer is "yes" and your new code uses the correct formatting :)